addons/point_of_sale/static/src/xml/Screens/PartnerListScreen/PartnerLine.xml

```diff
@@ -17,7 +17,7 @@
                 </div>
             </td>
             <td t-if="!env.isMobile">
-                <t t-esc="shortAddress" />
+                <t t-esc="props.partner.address" />
```

Comment:
@jotr-odoo thank you for you feedback, but if I got you right, I don't change anything in 16 I only fix in master ? because the PO is the one who told me to change this in 16

Reply:
No. You're also partially following what NLE said. First you're directly using `props.partner.address` in the template, so what's the point of keeping the method you modified aka `get address()` ? This is useless because it's called nowhere. Customization will not have overridden this method as well because you just "created it".
What NLE said was about keeping the method name as is it is in case some people had already overridden this in stable. So to not impact those potential (again, I hardly doubt there are but it's always best to follow good practices) override, you shouldn't just remove an used method like that in a template (unless there's an absolute necessity). 
Now how do you fix the issue for the other users ? You just change the content of the original method.
What I then said, was about master where you can do whatever you want like renaming a method or heck, even remove it to directly use the props in the template

Reply:
@trj-odoo ah my bad, I corrected it as you suggested